src/session.rs

```diff
@@ -2152,23 +2152,31 @@ impl Session {
     }
 
     /// Get the color at the given view coordinate.
-    fn color_at(&self, v: ViewId, p: ViewCoords<u32>) -> Rgba8 {
+    fn color_at(&self, v: ViewId, p: ViewCoords<u32>) -> Option<Rgba8> {
         let resources = self.resources.lock();
         let snapshot = resources.get_snapshot(&v);
 
-        assert!(snapshot.len() % std::mem::size_of::<Bgra8>() == 0);
```

Comment:
Any reason why this was removed?

Reply:
Because a snapshot being 1x1 has a len of 1, so this assertion makes no sense.

Reply:
Yeah - you're right. This is actually a regression - `len()` is supposed to return the number of bytes, which would mean it is always at least `4`.